v0.21/index.html

```diff
@@ -82,16 +82,11 @@
 
           
           <h1>Quick Start</h1>
-          
-          
-          <p><strong>Note</strong>: To run examples, please make sure that the flag <code>-Ypartial-unification</code><br />
-is enabled in your compiler options (i.e <code>scalacOptions += &quot;-Ypartial-unification&quot;</code> in sbt).
-This is enabled by default in the giter8 template.</p>
```

Comment:
It kind of stinks that this is only true for 2.13.  I wonder if we should leave a warning for our 2.12 users?

Reply:
`To run with 2.12 `...

Reply:
I like that idea.  Let's restore it with that language, and I think this is good to go.